specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -1913,6 +1913,252 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/continuouswebjobs": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "List continuous web jobs for an app, a specific deployment slot, and/or a specific scaled-out instance.",
+        "description": "List continuous web jobs for an app, a specific deployment slot, and/or a specific scaled-out instance.",
+        "operationId": "WebApps_ListContinuousWebJobs",
+        "produces": [
+          "application/json",
+          "text/json"
```

Comment:
It looks like all Antares (Azure Web Apps) APIs produce application/json, text/json, application/xml, and text/xml.  I believe this is by design.